src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -44,7 +44,11 @@ public abstract partial class Diagnostic : IEquatable<Diagnostic>, IFormattable
         /// </summary>
         /// <param name="descriptor">A <see cref="DiagnosticDescriptor"/> describing the diagnostic.</param>
         /// <param name="location">An optional primary location of the diagnostic. If null, <see cref="Location"/> will return <see cref="Location.None"/>.</param>
-        /// <param name="properties">An optional set of properties of the diagnostic. If null, <see cref="Properties"/> will return <see cref="ImmutableDictionary{TKey, TValue}.Empty"/>.</param>
+        /// <param name="properties">
+        /// An optional set of name-value pairs by means of which the the analyzer that creates the diagnostic
```

Comment:
Oh boy, I had to stare for a long time before I saw "the the". Nice catch :-) Will fix.